src/main.py

```diff
@@ -1,39 +1,30 @@
 from sanic import Sanic, response
+import models
 import os
 import json
 
 app = Sanic('Test API')
 
-DATABASE_FILE = 'database.json'
 
+@app.route('/store/<name>')
+async def r_store(req, name):
+	wid = models.WebID(
+		name=name,
+		uri='https://samvdkris.inrupt.net/profile/card#me'
```

Comment:
Correct me if I'm wrong but the WebID should be retrieved from the http request and not be hard coded.
You can maybe use a post request with a json body that contains the name and the WebID